Document 'Require Site Admins to Enable Two-factor Authentication' (#1999)
hashicorp/terraform-website: content/source/docs/enterprise/admin/general.html.md

-> **API:** See the [Admin Settings API](/docs/cloud/api/admin/settings.html).

## Require Site Admins to Enable Two-factor Authentication

> -> **Note**: This setting can be used in conjunction with SAML in Terraform Enterprise versions v202201-1 or later. Prior to this release, the setting had no effect if SAML was also enabled

Reviewer: ```suggestion
-> **Note**: You can use this setting in conjunction with SAML for Terraform Enterprise v202201-1 or later. For prior versions, this setting has no effect if SAML is also enabled.
```

> Enabling this setting adds the requirement that admins have two-factor authentication enabled to access site admin functionality, which can make the site more secure.

Reviewer: Just trying to make it easier to read :) I also wonder if we could start out with something other than using 'enabled" again. I removed that because it felt obvious, , but let me know if I've materially changed the meaning or you feel strongly we should open with that.

```suggestion
This setting can make the site more secure by requiring that admins enable two-factor authentication to access site admin functionality.
```

Author: Looks good to me (and easier to read)!

Admins that do not have two-factor authentication enabled may still log in, but will be unable to perform any admin-only functions until they enable and verify two-factor authentication.

> This setting can be used in conjunction with [SAML Single Sign On](https://www.terraform.io/docs/enterprise/saml/index.html).

Reviewer: ```suggestion
You can use this setting in conjunction with [SAML Single Sign On](https://www.terraform.io/docs/enterprise/saml/index.html).
```
